app/graphql/mutations/create_user.rb

```diff
@@ -0,0 +1,27 @@
+class Mutations::CreateUser < Mutations::BaseMutation
+  argument :name, String, required: true
+  argument :email, String, required: true
+  argument :phone, String, required: true
+  argument :bio, String, required: true
+  argument :location, String, required: true
+  argument :donor_status, Integer, required: true
+
+  field :user, Types::UserType, null: false
+  field :errors, [String], null: false
+
+  def resolve(name:, email:, phone:, bio:, location:, donor_status:)
+    user = User.new(name: name, email: email, phone: phone, bio: bio, location: location, donor_status: donor_status)
```

Comment:
⛏️ Consider putting each of the attributes onto a new line for easier grokkability.